packages/next/src/build/turbopack-build/impl.ts

```diff
@@ -155,6 +159,12 @@ export async function turbopackBuild(): Promise<{
       )
     }
 
+    if (!('routes' in entrypoints)) {
```

Comment:
we already accessed `routes` above on line 113 so we would have thrown there under this condition.

we should just re-order issue processing to happen immediately after calling `writeAllEntrypointsToDisk`